src/services/services.ts

```diff
@@ -345,6 +346,88 @@ namespace ts {
         }
     }
 
+    class TokenOrIdentifierObject implements Token {
+        public kind: SyntaxKind;
+        public flags: NodeFlags;
+        public parent: Node;
+        public jsDocComments: JSDocComment[];
+        public __tokenTag: any;
+
+        constructor(public pos: number, public end: number) {

```

Comment:
Sorry, [that's my fault](https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update3-vs). :)